CONTRIBUTING.md

```diff
@@ -431,6 +431,83 @@ After you have added new feature, to make sure it stays working, we recommend us
 
 [This](https://github.com/freeCodeCamp/chapter/contribute) is a good place to go if you are looking to get started.
 
+## Authorizing GraphGQL requests with `@Authorized` decorator
```

Comment:
I think it's better to be less specific, so it's more comfortable and easier for our future self to broaden it when necessary.

```suggestion
## Authorized GraphQL Requests
```